devel/development.ini.example

```diff
@@ -22,8 +22,6 @@ pyramid.debug_authorization = true
 pyramid.debug_notfound = true
 pyramid.debug_routematch = true
 pyramid.default_locale_name = en
-pyramid.includes =
```

Comment:
I think we should leave this, since it can be useful in the development environment.